fix: FPs in SQL query on multiple places

azurit opened the pull request:
For example typo (missing backtick) in SQL query caused FP:
```
`SELECT `k`.`ac FROM`
```



Fixes: https://github.com/coreruleset/phpmyadmin-rule-exclusions-plugin/issues/37

azurit committed:
Update phpmyadmin-rule-exclusions-before.conf

azurit committed:
Create 9513260.yaml

azurit committed:
Add PhpMyAdmin Rule Exclusions Plugin test case

EsadCetiner reviewed (approved):
Can you change the PR title to something more descriptive before merging?

azurit commented:
Done.

azurit committed:
Update phpmyadmin-rule-exclusions-before.conf

azurit committed:
Create 9513700.yaml

azurit committed:
Add disabled test for PhpMyAdmin Rule Exclusions

This test is disabled due to unsupported plugin configuration changes.

azurit committed:
Update phpmyadmin-rule-exclusions-before.conf

azurit committed:
Create 9513100.yaml

azurit committed:
Add rule removal for Referer header

azurit committed:
Add new test case for FP inside Referer header

azurit committed:
Add rule for preferences/features route

azurit committed:
Add ruleRemoveTargetById for import_file

azurit committed:
Update 9513700.yaml

azurit committed:
Comment out test case for compressed file import

Comment out a test case related to data import from compressed file.

azurit committed:
Update phpmyadmin-rule-exclusions-before.conf

azurit committed:
Remove SecRule for '/preferences/features' route

Removed specific SecRule for route '/preferences/features' from phpMyAdmin rule exclusions - fixed in CRS.

azurit commented on plugins/phpmyadmin-rule-exclusions-before.conf:
```diff
@@ -1073,4 +1083,12 @@ SecRule ARGS:route "@streq /" \
     ctl:ruleRemoveTargetById=951220;RESPONSE_BODY,\
     ctl:ruleRemoveTargetById=953100;RESPONSE_BODY"
 
+SecRule ARGS:route "@streq /preferences/features" \
```
Rule was completely removed as it was fixed directly in CRS.
